notebooks/ch11/gradient_convexity.md

```diff
@@ -30,73 +30,72 @@ tips['pcttip'] = tips['tip'] / tips['total_bill'] * 100
 
```

Comment:
In this file, you use "mean absolute error", but I think you can abbreviate with MAE esp. when in parallel with MSE. We can ask @SamLau95  if you should reintroduce that MAE is mean absolute error. I do it in Ch10 already.

Reply:
Also for this file, thetas were changed to \hat{\theta} instead of \hat{\theta_0}